Prac_05/emails.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict, Any
+
+
+def get_name_from_email(email):
+    name = email.split("@")
+    name = name[0]
+    if "." in name:
+        name = name.split(".")
+        name = " ".join(name)
+
+    return str(name).title()
+
+
+
+EMAILS_AND_NAMES = {}
+stop = False
+while not stop:
+    email = input("Email: ")
+    if email == "":
```

Comment:
This is not the standard while loop pattern. Can you see how this is "clunky"? We've never done it this way.